packages/valory/skills/market_manager_abci/bets.py

```diff
@@ -34,6 +35,46 @@
 BINARY_N_SLOTS = 2
 
 
+class QueueStatus(Enum):
+    """The status of a bet in the queue."""
+
+    # Common statuses
+    EXPIRED = -1  # Bets that have expired, i.e., the market is not live anymore
+    FRESH = 0  # Fresh bets that have just been added
+    TO_PROCESS = 1  # Bets that are ready to be processed
+    PROCESSED = 2  # Bets that have been processed
+    REPROCESSED = 3  # Bets that have been reprocessed
+
+    def is_fresh(self) -> bool:
+        """Check if the bet is fresh."""
+        return self == QueueStatus.FRESH
+
+    def is_expired(self) -> bool:
+        """Check if the bet is expired."""
+        return self == QueueStatus.EXPIRED
+
+    def move_to_process(self) -> "QueueStatus":
+        """Move the bet to the process status."""
+        if self == QueueStatus.FRESH:
+            return QueueStatus.TO_PROCESS
+        return self
+
+    def move_to_fresh(self) -> "QueueStatus":
+        """Move the bet to the fresh status."""
+        if self != QueueStatus.EXPIRED:
+            return QueueStatus.FRESH
+        return self
+
+    def next_status(self) -> "QueueStatus":
+        """Get the next status in the queue."""
+        if self == QueueStatus.TO_PROCESS:
+            return QueueStatus.PROCESSED
+        elif self == QueueStatus.PROCESSED:
+            return QueueStatus.REPROCESSED
+        else:
+            return QueueStatus.FRESH
```

Comment:
How does a bet move from `FRESH` to `TO_PROCESS`?

Reply:
when all the bets, that are present in the list of bets that are FRESH or not EXPIRED.